test-upgrade/2to3/2to3.js

```diff
@@ -81,6 +84,13 @@ contract("Colony contract upgrade", () => {
       await colony.finishUpgrade2To3();
     });
 
+    it("should be able to assign arbitration role", async function() {
+      let hasRole = await colony.hasUserRole(accounts[3], 2, 2);
+      await colony.setArbitrationRole(1, 0, accounts[3], 2, true, { from: TESTCOLONY_OWNER });
+      hasRole = await colony.hasUserRole(accounts[3], 2, 2);
```

Comment:
Nitpick but it would slightly more readable if there was a `const ARBITRATION_ROLE = 2` somewhere -- `colony.hasUserRole(accounts[3], 2, 2)` is hard to parse :(

Reply:
Sure, we have a constant for that in the helper.